### ISSUE TITLE
Add an option for buildall to build on deb- even if it detects other systems.

### ISSUE THREAD

**Issue by blue-devil-93** [rozniak/xfce-winxp-tc#355]
Tried to build your Scripts on asmi linux. Which is a small xubuntu based iso image. It will detect rpm. Had to disable the detect os sh a bit and then it compiles nicely. No issue here but can you please add a switch to skip operatig system check for the build scripts if an user knows that deb is the wanted package architecture?

**Comment by rozniak** [comment]
Yeah there is #346 which is the same thing as this, I just haven't gotten around to it... slight complication has been `chkdeps.sh` which is meant as a sort of non-interactive thing.

NTS: Look into this - `chkdeps.sh` should only output package names to stdout (get rid of 'all dependencies verified') and output clashing package managers to stderr? (if detecting clashes isn't going to be a hassle)

**Comment by rozniak** [comment]
Sorry it has taken so long for me to get to this, just doing some final testing.

Once merged it will be possible to specify `./buildall.sh -t deb` to override the default behaviour.
